input/pagecontent/conformance.md

```diff
@@ -37,10 +27,12 @@ The [Profiles and Extensions](profiles-and-extensions.html) page lists the AU eR
 
 AU eRequesting profile elements include mandatory and *Must Support* requirements. [Mandatory elements](#mandatory-elements) are required and have a minimum cardinality of 1 (min=1). [Must Support](#must-support-and-obligation) elements have defined conformance obligations in AU eRequesting based on actor roles.
 
+The [Actors and Capabilities](capability-statements.html) page list the AU eRequesting Actor definitions and their CapabilityStatements.
```

Comment:
It seems that the resolution description for section 2.1.1 AU eRequesting Profiles and Extensions is sort of overwritten by [FHIR-46812](https://jira.hl7.org/browse/FHIR-46812), as the Profiles and Extensions page now lists not only the AU eRequesting extensions defined for this implementation guide but also referenced extensions.

Reply:
yes we should also make that update related to FHIR-46812 in this branch

Reply:
Amended, thanks.